interface/patient_file/summary/create_portallogin.php

```diff
@@ -165,13 +170,22 @@ function filterTwigTemplateData($twig, $pid, $templateName, $data)
         error_log('OpenEMR Error : OpenEMR is not working because unable to create a hash.');
         die("OpenEMR Error : OpenEMR is not working because unable to create a hash.");
     }
-    array_push($query_parameters, $hash);
 
+    array_push($query_parameters, $hash);
     array_push($query_parameters, $pid);
+
+    EventAuditLogger::instance()->newEvent(
+        "patient access",
+        $_SESSION['authUser'],
+        "Default",
```

Comment:
also, here, best to use $_SESSION['authProvider']
(if we ever in future make use of that group thingy)